src/handler.js

```diff
@@ -114,11 +116,45 @@ const handle = async message => {
       Authorization: `Bearer ${process.env.ACCESS_TOKEN}`,
     },
   }).then(response => {
-    response.data.pipe(resizer).pipe(writeStream);
+    response.data.pipe(resizer).pipe(resizedStream);
+  });
+
+  let b64Chunks = [];
+
+  resizedStream.pipe(b64Stream);
```

Comment:
Might be worth keeping this with the other pipes so you can see all of the stream outputs in one bit of the code rather than this one deciding to pipe/listen a bit later on? Just a thought.